docs/content/installation/install_hostdefiner.md

```diff
@@ -0,0 +1,13 @@
+# Installing the host definer
+
+Install the HostDefiner custom resource to enable dynamic host definitions of the CSI driver nodes.
```

Comment:
Also I am not sure regarding the phrase "dynamic" I think "automatic" will more suit.

Reply:
this was discussed previously with @oriyarde  and @erantzabari  and they preferred "dynamic" to automatic.

Reply:
@rivkap if this has been discussed so this discussion is closed